shinydashboard/lantern/modules/downloadsmodule.R

```diff
@@ -55,6 +62,27 @@ downloadsmodule <- function(
     }
   )
 
+  # Downloadable csv of selected dataset
+  output$download_data <- downloadHandler(
```

Comment:
I think we should also include the CSV Field Description file here too, just for consistency